core/scripts/local/RequestOraclePrice.js

```diff
@@ -74,8 +75,19 @@ const runRequestOraclePrice = async function(callback) {
   }
 
   const finder = await Finder.deployed();
-  await run(finder, argv.identifier, argv.time);
-  callback();
+
+  const callRun = async () => {
+      await run(finder, argv.identifier, argv.time);
+      callback();
+    };
+
+  // Note: use ENV for port because in some cases GCP doesn't allow the user to change the docker args.
+  if (process.env.PORT) {
+    // Only trigger on request if PORT is in the ENV. Otherwise, we assume the user wants it called syncrhonously.
```

Comment:
typo in "syncrhonously"

Reply:
Haha, I don't think I've ever typed that word correctly on the first try. Nice catch